configure

```diff
@@ -0,0 +1,625 @@
+#  ! Variable definitions                                                !
+#  !---------------------------------------------------------------------!
+
+# quick version flags
+serial='no'
+mpi='no'
+cuda='no'
+cudampi='no'
+debug='no'
+installer=''
+exename=''
+
+# keeps track of what compilers being used
+fort='no'
+cc='no'
+cxx='no'
+nvcc='no'
+
+# where to install
+useprefix='false'
+quick_prefix="$QUICK_HOME"
+
+# flag to indicate if a so library compilation is required
+shared='no'
+
+# compiler specific flags
+fort_flags=''
+cc_flags=''
+cxx_flags=''
+cuda_flags=''
+cpp_flags=''
+
+# archiver related variables
+arch='ar'
+arch_flags='cr'
+ranlib='ranlib'
+libext='a'
+
+# linker flags
+ldflags=''
+cflags=''
+
+# optimization levels
+opt_flags='-O2'
+blas_opt_flags='-O3'
+
+# cuda architecture flags
+uspec_arch='false'
+cuda_arch=''
+cuda_arch_flags=''
+
+# extra cuda flags
+cuda_ext_flags='-Xptxas=-v -m64 -use_fast_math'
+
+# cuda include flags
+cuda_incl_flags=''
+
+# flags to generate PIC in so library case 
+lib_flags=''
+cuda_lib_flags=''
+
+# debug flags
+fort_debug_flags=''
+cc_debug_flags=''
+cxx_debug_flags=''
+cuda_debug_flags=''
+
+#  !---------------------------------------------------------------------!
+#  ! Check user input                                                    !
+#  !---------------------------------------------------------------------!
+if [ $# -lt 1 ]; then print_help; fi
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -serial)      serial='yes';mpi='no';cuda='no';cudampi='no';;
+    -mpi)         serial='no';mpi='yes';cuda='no';cudampi='no';;
+    -cuda)        serial='no';mpi='no';cuda='yes';cudampi='no';;
+    -cudampi)     serial='no';mpi='no';cuda='no';cudampi='yes';;
+    -debug)       debug='yes';;
+    -shared)      shared='yes';; 
+    -arch)        shift; cuda_arch=$1; uspec_arch='true';;
+    --prefix)     shift; quick_prefix=$1; useprefix='true';;
+    -h| -H| -help| --help) print_help;;
+    -*)  echo "Error: $1 is an unknown flag"
+         echo "       Run this script as './configure [flags] compiler'"
+         echo "       Run './configure --help' for more information"
+         exit 1;;
+
+     *)  if [ $# -gt 1 ]; then
+           echo "Error: $1 is an unknown compiler"
+           echo "       Run this script as './configure [flags] compiler'"
+           echo "       Run './configure --help' for more information"
+           exit 1
+         fi
+
+         compiler=$1;;
+  esac
+  shift
+done
+
+# check prefix and perform necessary operations 
+if [ "$useprefix" = "true" ]; then
+  if [ "$QUICK_HOME" = "$quick_prefix" ]; then
+    echo "Specified --prefix and the current folder is the same. Please choose a different location."
+    exit 1
+  fi
+
+  # create tests and basis directories if they dont exist
+  if [ ! -d "$quick_prefix/test" ]; then
+    mkdir -p "$quick_prefix/test"
+    echo "Creating symlink for $quick_prefix/test"
+    cd "$quick_prefix/test"
+    lndir "$QUICK_HOME/test"
```

Comment:
`lndir` is by default not available on CentOS. We should not rely on commands that must be installed with user packages.

Reply:
I have fixed this in commit fd802a5eae249b80c9852d3ef284124277225cf9.

Reply:
This is fixed. Resolving conversation.